workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -659,6 +682,8 @@ def __iter__(
 
         for i in range(self._fv3gfs.get_step_count()):
             diagnostics: Diagnostics = {}
+            # clear the state updates in case some updates are on intervals
```

Comment:
If this is not cleared, the previous time steps' state updates will persist and be applied until they are updated again at the next interval.

Reply:
We don't intentionally persist state updates to the next timestep for anything do we? I know we used to do this for momentum tendencies but I think that is gone now that we update wind on the D grid. So it seems like this is fine.